scripts/helpers/ts-api.js

```diff
@@ -0,0 +1,54 @@
+const request = require('request-promise-native');
+
+class TSAPIError extends Error {
+	constructor(...args) {
+		super(...args);
+		this.name = this.constructor.name;
+		Error.captureStackTrace(this, TSAPIError);
+	}
+}
+
+const doRequest = async (auth, { body, method = 'GET', url }) => {
+	const options = {
+		body,
+		gzip: true,
+		headers: {
+			'user-agent': 'bucko-app',
+			'x-tradeshift-tenantid': auth.companyId,
```

Comment:
I normally see this as something like `X-Tradeshift-TenantID`, does capitalization matter?

Reply:
headers are case insensitive per the http spec - https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html